Dockerfile

```diff
@@ -1,5 +1,11 @@
 FROM ocaml/opam:ubuntu-20.04-ocaml-4.14
 
+ARG OPAM_JOBS=0
```

Comment:
Is this something we want the user to choose with a build-arg or just use `nproc`?

Reply:
Hmmm, that's not a bad idea. Let me experiment with that.